tests/functional/centos/7/shrink_osd/group_vars/all

```diff
@@ -1,9 +1,11 @@
 ---
 ceph_origin: repository
 ceph_repository: community
+cluster: ceph
 public_network: "192.168.71.0/24"
 cluster_network: "192.168.72.0/24"
-radosgw_interface: eth1
+monitor_interface: eth1
```

Comment:
`monitor_address` is already defined in inventory host file, why do we need to set `monitor_interface` here ?

Reply:
@guits I removed this, as per my comment above to leseb. the radosgw change was needed to resolve an error (shown in the git commit), though perhaps its not the root cause